### ISSUE TITLE
Reconstructed-mandible vessel position

### ISSUE THREAD

**Issue by mauigna06** [SlicerIGT/SlicerBoneReconstructionPlanner#53]
I saw sometimes users have a model of the fibula vessel.

![vesselOfFibula](https://user-images.githubusercontent.com/19158307/140645580-e185aa65-fda9-494d-8589-268f40d683f0.png)

So I suppose it is important on planning to see it's relative position/orientation regarding the fibula.

I could show the vessel parts that correspond to the reconstructed fibula pieces in the position/orientation they would be if the fibulaPlanes cutted the whole leg to make the reconstruction (relative position/orientation of the vessel pieces regarding the fibula pieces is conserved).

Would this be useful? @cmfsx, @mrtig

**Comment by mrtig** [comment]
Yes, it’s good to have it for reference. It’s important to know where the vessels enter the fibula. When planning the last case I thought it might be good to have a model that has both the vessels and the bone so it would be possible to see all the entry points.
I will make a post about it next week.

**Comment by cmfsx** [comment]
@mauigna06 We usually segment it during our planning mainly for visualization.  What is important is not really the peroneal artery but rather the location of the skin perforators. Because it is what dictates the location of the skin paddle.  These perforators are not visible on CECT. We have thought of how to incorporate this information into the planning.  One way is to do the doppler before the CECT and put radiopaque markers on the skin. So we can locate them in the CT. We have not yet done that as an experiment.  


![Screenshot_1](https://user-images.githubusercontent.com/57766971/140646536-cbe7036d-4887-42e5-9889-b4217117efee.png)
. 

> I could show the vessel parts that correspond to the reconstructed fibula pieces in the position/orientation they would be if the fibulaPlanes cutted the whole leg to make the reconstruction

I don't think it will really make any sense other than the visual appeal.

**Comment by mauigna06** [comment]
> I saw sometimes users have a model of the fibula vessel.
> 
> So I suppose it is important on planning to see it's relative position/orientation regarding the fibula.
> 
> I could show the vessel parts that correspond to the reconstructed fibula pieces in the position/orientation they would be if the fibulaPlanes cutted the whole leg to make the reconstruction (relative position/orientation of the vessel pieces regarding the fibula pieces is conserved).

I think we haven't reach conclusion on this, I've seen some surgeons using the vessel visualization but I don't know why

**Comment by SteveMaisi** [comment]
it is used for visualization
1. To see the length of peroneal vessels in relation to the last osteotomy cut of the fibula proximally. this can estimate the length of the artery left for anastomosis purposes (for example to do anastomosis ipsilateral or contralateral neck, anterior or posterior)
2. to see its courses. some vessels run along the posterior inferior aspect of the fibula. but not all the same. May alter surgical dissection approach.
3.  to see anatomy variations and abnormalities.

**Comment by mauigna06** [comment]
> it is used for visualization
> 
>     1. To see the length of peroneal vessels in relation to the last osteotomy cut of the fibula proximally. this can estimate the length of the artery left for anastomosis purposes (for example to do anastomosis ipsilateral or contralateral neck, anterior or posterior)
> 
>     2. to see its courses. some vessels run along the posterior inferior aspect of the fibula. but not all the same. May alter surgical dissection approach.
> 
>     3. to see anatomy variations and abnormalities.

Related https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner/issues/111
